test/Lachain.CryptoTest/ChainIdTest.cs

```diff
@@ -290,6 +292,8 @@ public byte[] RecoverSignatureHashed(byte[] messageHash, byte[] signature, bool
                 var pk = new byte[64];
                 var encodedRecId = RestoreEncodedRecIdFromSignatureBuffer(signature);
                 var recId = (encodedRecId - 36) / 2 / ChainId(useNewChainId);
+                if (recId < 0 || recId > 3)
```

Comment:
I think if we can add `recId >= 0 && recId <= 3` here as well it is much readable, Because we are throwing exception in different format and both of them means same